scripts/funcotator/data_sources/gnomAD/indexFeatureFile.wdl

```diff
@@ -0,0 +1,102 @@
+# Run IndexFeatureFile on a VCF file. 
```

Comment:
A single VCF file?  It looks like multiple VCF files....

Is this script supported?

Reply:
Old documentation.  Fixed!

Yes.  I want to maintain a set of files that will be able to generate lifted-over VCFs for the purposes of gnomAD and others.